hackathon_site/dashboard/frontend/src/components/general/Footer/Footer.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Footer from "./Footer";
+
+describe("Footer", () => {
+    test("renders", () => {
+        const { asFragment } = render(<Footer />);
+        expect(asFragment()).toMatchSnapshot();
```

Comment:
Since the snapshot has the year in it, this test will fail in 2021. You'll need to mock the date for all tests. See below for a shorter way to mock the date. You may also want to do it outside of the `test` functions (but within `describe`, that way it will apply to all of these tests, then you can just override it for the ones you need it to be a different date.

Reply:
so I would do a snapshot, mock the function and do 2 tests for each case where one test would be 2020 and the other one where the year is not 2020?

Reply:
You need to mock the date function before you render the component; so you should be mocking it for all tests.

See below, technically if you mock the year as 2020 in this test, you've covered the case where the date is 2020 and just need the case where it is not 2020 below. 

Up to you if you'd like to make the 2020 case explicit like the one below, it's not strictly necessary as long as you mock the date as 2020 here.